### PR TITLE
Update trade events: ExecuteTrade(proxy) -> user facing data.  KyberTrade (network) -> internal trade details

### PR DESCRIPTION
**Pull request by ayobuenavista**

### CODE REVIEW

File: test/sol6/kyberNetwork.js [hunk 1]
```diff
@@ -1207,6 +1202,28 @@ contract('KyberNetwork', function(accounts) {
                 for (let i = 0; i < expectedResult.e2tIds.length; i++) {
                     Helper.assertEqual(expectedResult.e2tIds[i], actualE2Tids[i], "E2T id not equal");
                 }
+
+                let actualT2EsrcAmounts = txResult.logs[3].args.t2eSrcAmounts;
+                let actualE2TsrcAmounts = txResult.logs[3].args.e2tSrcAmounts;
+                Helper.assertEqual(expectedResult.t2eSrcAmts.length, actualT2EsrcAmounts.length, "T2E srcAmounts length not equal");
```
Comment (Anyhowclick) [inline test/sol6/kyberNetwork.js:1208]: Helper.assertArrayEqual(expectedResult.t2eSrcAmts, actualT2EsrcAmounts);
Helper.assertArrayEqual(expectedResult.e2tSrcAmts, actualE2TsrcAmounts);

Comment (tranvictor) [reply test/sol6/kyberNetwork.js:1205]: Why do we use "Amts" and "Amounts" to express the same meaning of "amount"?

Comment (ilanDoron) [reply test/sol6/kyberNetwork.js:1205]: I think we should change Amt --> Amounts.
its plural here since its an array.

Comment (ayobuenavista) [reply test/sol6/kyberNetwork.js]: Amts changed to Amounts. Commit still squashed to one.

**Pull request merged**